linux_os/guide/system/accounts/accounts-pam/set_password_hashing_algorithm/set_password_hashing_algorithm_logindefs/rule.yml

```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: fedora,ol7,ol8,rhel7,rhel8,rhv4,sle12,sle15,ubuntu2004,wrlinux1019
+prodtype: fedora,ol7,ol8,rhel7,rhel8,rhel9,rhv4,sle12,sle15,ubuntu2004,wrlinux1019
```

Comment:
This is a classical textfilecontent rule which should work anywhere. I will check it out.

Reply:
So it turns out that the check on Fedora fails because the rule is not applicable according to `shared/checks/oval/installed_env_has_login_defs.xml` as it is under the `login_defs` platform.